File: abc/WhenIsWeekend/Lambda.go
package main

import (
    "fmt"
    "time"
)

func main() {
    fmt.Println("When’s Weekend?")

    today := time.Now().Weekday()

    switch today {
    case time.Sunday:
        fmt.Println("Today.")
    case time.Saturday:
        fmt.Println("Today.")
    case time.Friday:
        fmt.Println("Tomorrow.")
    case time.Thursday:
        fmt.Println("In two days.")
    default:
        fmt.Println("Too far away.")
    }
}

File: abc/WhenIsWeekend/Lambda.rb
if __FILE__ == $PROGRAM_NAME
    puts("When’s Weekend?")

    today = Time.now().wday

    case today
    when 0
        puts("Today.")
    when 6
        puts("Today.")
    when 5
        puts("Tomorrow.")
    when 4
        puts("In two days.")
    else
        puts("Too far away.")
    end
end
